packages/matplotlib/src/html5_canvas_backend.py

```diff
@@ -214,6 +215,8 @@ def set_joinstyle(self, js):
             raise ValueError('Unrecognized join style. Found {0}'.format(js))
 
     def set_linewidth(self, w):
+        if w == 0:
+            self.stroke = False
```

Comment:
We also need to make sure stroke is `True` if `w != 0`.  So maybe just:

```
self.stroke = (w != 0)
```

Reply:
Aah yes, doing, thanks!